File: src/main/java/fr/lewon/web/bot/entities/Cost.java
package fr.lewon.web.bot.entities;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Cost {

	@JsonProperty("$")
	private Integer regularCost;

	@JsonProperty("HC")
	private Integer specialCost;

	@JsonProperty("*")
	private Integer energyCost;


	public Integer getRegularCost() {
		return regularCost;
	}

	public void setRegularCost(Integer regularCost) {
		this.regularCost = regularCost;
	}

	public Integer getSpecialCost() {
		return specialCost;
	}

	public void setSpecialCost(Integer specialCost) {
		this.specialCost = specialCost;
	}

	public Integer getEnergyCost() {
		return energyCost;
	}

	public void setEnergyCost(Integer energyCost) {
		this.energyCost = energyCost;
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/diverse/Cost.kt
package fr.lewon.bot.hh.entities.diverse

import com.fasterxml.jackson.annotation.JsonProperty

class Cost {
    @JsonProperty("$")
    var regularCost: Int? = null
    @JsonProperty("HC")
    var specialCost: Int? = null
    @JsonProperty("*")
    var energyCost: Int? = null

}